docs/guides/runtime-options-overview.mdx

```diff
@@ -18,288 +19,298 @@ service = QiskitRuntimeService(channel="ibm_cloud", channel_strategy="q-ctrl")
+|                      | init_qubits              |                         | `True`/`False`                                                                         | `True`                    |              |
+|                      | shots                    |                         | Integer no larger than `backend.max_shots`.                                                                   | 4000|              |
+| [environment](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.EnvironmentOptions)          |                          |                         |           |                                  |  environment  |
+|                      | log_level                |                         |`DEBUG`/`INFO`/`WARNING`/`ERROR`/`CRITICAL`                                          |`WARNING`                          |              |
+|                      | callback                 |                         |Callable function that receives Job ID and Job result.                                 |`None`                           |              |
+|                      | job_tags                 |                         |List of tags                                                                           |`None`                           |              |
+| [simulator](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.SimulatorOptions)            |                          |                         |                        |                     |   simulator  |
+|                      | noise_model              |                         |   [Qiskit Aer NoiseModel](build-noise-models), or its representation        | `None`                      |              |
+|                      | seed_simulator           |                         |   integer                                                                               | `None`                    |              |
+|                      | coupling_map             |                         |  List of directed two-qubit interactions                                          | full connectivity                   |              |
+|                      | basis_gates              |                         |  List of basis gate names to unroll to          | The set of all basis gates supported by [Qiskit Aer simulator](https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.AerSimulator.html) |              |
 
-```python
-from qiskit_ibm_runtime import Estimator, Options
+  </TabItem>
 
-options = Options()
-options.transpilation.skip_transpilation = True
 
-estimator = Estimator(backend=backend, options=options)
-```
 
-For more information and a complete list of advanced transpilation options, see the Advanced transpilation options table in the
-[Runtime compilation topic](./configure-error-suppression#transpilation-table).
+</Tabs>
 
-### Error mitigation
 
+<span id="options-compatibility-table"></span>
+### Options compatibility
```

Comment:
Note that PEA has not been publicly released, it's still an experimental option. It is planned to be made public at the end of July.

Reply:
@jyu00 - should we pull all the PEA content until July?  I have an issue for that, so I could move this PEA content in there.

Reply:
Jessie said that it's OK to keep here because it is already publicly documented.